src/Exceptions/HttpPayloadTooLarge.php

```diff
@@ -8,4 +8,3 @@ class HttpPayloadTooLarge extends HttpException {
     public const DESCRIPTION = "Indicates that the server is refusing to process a request because the request payload is larger than the server is willing or able to process.";
     public const HREF = "https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/413";
 }
-
```

Comment:
Removed unix line ending - ALWAYS have a unix line ending, it can cause very weird issues.

Just use https://marketplace.visualstudio.com/items?itemName=jkiviluoto.tws you can use this to add them automatically. It also removes trailing whitespaces